superset/assets/version_info.json

```diff
@@ -0,0 +1 @@
+{"GIT_SHA": "8ebedbbf4da40e62313420afe93bdc811d024b38", "version": "0.999.0dev"}
```

Comment:
This SHA checked into source control seems questionable. How will this be kept in sync? Was this committed in error? Also note that this directory has moved to `superset-frontend`

Reply:
this is an error - will clean up and rebase again